modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -58,13 +58,23 @@ You cannot have 2 indexes with the same name inside the same bucket and scope.
 . Expand *Mappings* and create a new type mapping on a collection:
 .. Click btn:[Add Type Mapping].
 .. In the *Collection* list, select the collection that contains the documents you want to include in your index. 
-.. Select *Only index specified fields*. 
+.. Select the [.ui]*only index specified fields* checkbox.
+.. Unset the [.ui]*#default* field
```

Comment:
```suggestion
.. Select *only index specified fields*.
.. Clear *#default*.
```

Reply:
@RayOffiah this is non-negotiable, please make these changes, don't just mark them as resolved.

Reply:
Ah, my mistake; I think I miskeyed; I was going to accept the change, but now I think about, I think I prefer my version.

The reason DOC-13488 was filed in the first place was that the filer(?) thought that the original text was asking him to look a set of specified fields: 

`Regarding  these instructions - it says "Only index specified fields".   What fields? Specified where?`

I would prefer to make it clear that he's looking for a checkbox.

But I can remove the [.ui] attributes.